src/pybind/mgr/dashboard/frontend/src/testing/unit-test-helper.ts

```diff
@@ -18,29 +18,26 @@ import {
   AlertmanagerNotificationAlert,
   PrometheusRule
 } from '../app/shared/models/prometheus-alerts';
-import { _DEV_ } from '../unit-test-configuration';
-
-export function configureTestBed(configuration: any, useOldMethod?: boolean) {
-  if (_DEV_ && !useOldMethod) {
-    const resetTestingModule = TestBed.resetTestingModule;
-    beforeAll((done) =>
-      (async () => {
-        TestBed.resetTestingModule();
-        TestBed.configureTestingModule(configuration);
-        // prevent Angular from resetting testing module
-        TestBed.resetTestingModule = () => TestBed;
-      })()
-        .then(done)
-        .catch(done.fail)
-    );
-    afterAll(() => {
-      TestBed.resetTestingModule = resetTestingModule;
-    });
-  } else {
-    beforeEach(async(() => {
+
+export function configureTestBed(configuration: any) {
+  const testBedApi: any = getTestBed();
+  const resetTestingModule = TestBed.resetTestingModule;
+  beforeAll((done) =>
+    (async () => {
+      // prevent Angular from resetting testing module
+      TestBed.resetTestingModule = () => TestBed;
+      TestBed.resetTestingModule();
       TestBed.configureTestingModule(configuration);
-    }));
-  }
+    })()
+      .then(done)
+      .catch(done.fail)
+  );
+  afterEach(() => {
+    testBedApi._instantiated = false;
```

Comment:
We might need more code here.
As you can see in [ng-bullet](https://github.com/topnotch48/ng-bullet-workspace/blob/master/projects/ng-bullet/src/testing.ts#L29) they have 2 more calls in afterEach.
the Ivy one will be needed when we update to angular 9, so please add that one.
the 1st one, I'm not sure.

Alternatively we can just use ng-bullet here. We would still call `configureTestBed `, but instead of having our logic here, we would just call ng-bullet.

If we don't go with ng-bullet, please remove it from package.json and update the unit test file that uses it.

Reply:
Seems like ng-bullet does a view more optimizations than only the mentioned running it overall tests to compare it took only 2m 4s 852ms where the current solution took 6m 48s 716ms, even if it was faster running single test suites.

Therefore we use `ng-bullet` know :)